pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -879,15 +879,15 @@
       "InboxOn": {
         "netcoreapp2.0": "4.2.0.0",
         "netcoreapp2.1": "4.2.1.0",
-        "netcoreapp3.0": "4.2.2.0",
+        "netcoreapp3.0": "4.3.0.0",
         "net45": "4.0.0.0",
         "net46": "4.0.10.0",
         "portable46-net451+win81": "4.0.0.0",
         "portable45-net45+win8": "4.0.0.0",
         "monoandroid10": "Any",
         "monotouch10": "Any",
         "uap10.0.16299": "4.2.1.0",
-        "uap10.0.16300": "4.2.2.0",
+        "uap10.0.16300": "4.3.0.0",
```

Comment:
This fixed the last error with UWP. Now that we have bumped the minor version, there shouldn't be anything pointing to 4.2.2.0.